### PR TITLE
chkdsk: add page

### PR DESCRIPTION
**Pull request by jpiamjariyakul**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/windows/chkdsk.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# chkdsk
+
+> Checks file system & volume metadata for errors.
+> More information: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/chkdsk/>.
```
Comment (navarroaxel) [inline pages/windows/chkdsk.md:4]: We don't need i18n in the MSFT's URLs:
```suggestion
> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/chkdsk>.
```

File: pages/windows/chkdsk.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# chkdsk
+
+> Checks file system & volume metadata for errors.
```
Comment (navarroaxel) [inline pages/windows/chkdsk.md:3]:
```suggestion
> Check file system and volume metadata for errors.
```

File: pages/windows/chkdsk.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Specify the drive letter (followed by a colon), mount point, or volume name to check:
+
+`chkdsk {{volume}}`
+
+- Fix errors on specified volume:
+
+`chkdsk {{volume}} /f`
+
+- Dismount specified volume before checking:
```
Comment (navarroaxel) [inline pages/windows/chkdsk.md:14]:
```suggestion
- Dismount a specific volume before checking:
```

File: pages/windows/chkdsk.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# chkdsk
+
+> Checks file system & volume metadata for errors.
+> More information: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/chkdsk/>.
+
+- Specify the drive letter (followed by a colon), mount point, or volume name to check:
+
+`chkdsk {{volume}}`
+
+- Fix errors on specified volume:
```
Comment (navarroaxel) [inline pages/windows/chkdsk.md:10]:
```suggestion
- Fix errors on a specific volume:
```

File: pages/windows/chkdsk.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`chkdsk {{volume}} /f`
+
+- Dismount specified volume before checking:
+
+`chkdsk {{volume}} /x`
+
+- Use with NTFS only; change the log file size to the specified size:
```
Comment (navarroaxel) [inline pages/windows/chkdsk.md:18]: Maybe this is more clear:
```suggestion
- Change the log file size to the specified size (only for NTFS):
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @jpiamjariyakul, thanks for your submission. I left some comments.

**Review by jpiamjariyakul — COMMENTED** [review]
Thanks @navarroaxel! I've applied the edits per your suggestions.

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]
Brilliant! Thanks @jpiamjariyakul, I'm surprised this hadn't already been added. 👍🏻

**Pull request merged**